### ISSUE TITLE
Masked MSE loss

### ISSUE THREAD

**Issue by njellinas** [NVIDIA/tacotron2#244]
Since the input to the decoder are batched sequences of different lenghts, shouldn't there be a masked MSE loss, not taking into account the padding frames and the extra frames that are produced after the true sentence length?

**Comment by rafaelvalle** [comment]
Yes! This is something in our to-do list. Please put a PR if you decide to do so.

**Comment by njellinas** [comment]
Well I finally found it in another repository (Mozilla's TTS) :P
Is it possible to include this code in your repo or not? (because of license implications)

**Comment by rafaelvalle** [comment]
Yes, Mozilla's license is very permissive. 
https://github.com/mozilla/TTS/blob/master/LICENSE.txt
Just put the PR and will review it.
